.github/workflows/galaxy.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: Release to Ansible Galaxy
```

Comment:
This file is used to publish Ansible role to Ansible Galaxy.
Same as following code in .travis.yml
```
notifications:
  webhooks: https://galaxy.ansible.com/api/v1/notifications/
```